Automattic/jetpack
# Search: use correct link for "learn more" help icon links in /search …

Fixes #4920

To Test: 
- Search for `photon`
- Click the little help icon 
- Should take you to https://jetpack.com/support/photon
